### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by jaredramirez** [wezterm/wezterm#969] [General]
Hey there,

I use Nix to install Wezterm and the last release was in May. I'd love to be able to use a handful of the things that've been merged since then! Does anyone have any idea when there will be a new release?

**Comment by wez** [comment]
I think we're getting close: I'd made some changes that hurt window event dispatching performance but recently resolved those. If things are looking generally good, then I might cut a release this weekend. I'd appreciate hearing feedback about how we're shaping up vs. the prior release: I generally live on a build that is < 1 week from the tip of `main` so I don't have that frame of reference!

**Reply by jaredramirez** [reply]
For me, the main things I'm looking forward too in are:

* Fixed: Windows: first character after wezterm ssh keyboard authention was swallowed #771
* Fixed: closing last pane in a tab via CloseCurrentPane could cause the window to close #890
* Fixed: splitting panes in multiplexer could fail after a network reconnect #781
* Fixed: multiplexer now propagates toast notifications and color palette to client #489 #748

My favorite & primary use of Wezterm (outside of normal, day-to-day things) is using it as a multiplexer for my home server. It's far and away the best solution for working comfortably on a remote machine. I'm super happy with pretty much everything else. Thanks for the great piece of software!

**Comment by jaredramirez** [comment]
Any update on this?

**Reply by wez** [reply]
https://wezfurlong.org/wezterm/changelog.html#20210814-124438-54e29167

**Comment by friederbluemle** [comment]
Any chance of a new release some time soon? So many updates since the last release :heart_eyes: 

Thanks!!

**Reply by wez** [reply]
Getting close: there's some work I want to wrap up around the tab bar presentation before making the next release.

I'm moving out of state this month, and I'm not sure if I'll have time to make that good enough before I move, so we may be looking at next month before a release happens.

You should be able to use the nightly builds in the mean time though!

**Reply by friederbluemle** [reply]
Sounds good, good luck with the move!
